### DISCUSSION TITLE
Flickering/stuttering progress-bar

### DISCUSSION THREAD

**Discussion by Stianlars1** [katspaugh/wavesurfer.js#3434] [Q&A]
After my update to the latest version 7.5.3 i now have some flickering in my progress-bar. It flickers black. And at first i thought maybe it was something weird with my audio state and if my app had some weird delay-issues or some other lags/faults for creating this stutter/flicker. 

Its really weird, it has never happened before, and i havent done anything spectacular to my code. Just to be sure that nothing form my side was wrong, i have reverted back to older versions just to see if it were still happening (to a time i know for a fact it did work). And it still stutters. 

Maybe im stupid, but have anyone els ehad problems with stutter/flicker of the progress bar?

Ive added the screen-recording for example of the problem. 

I cant figure out why it does this. Help?

https://github.com/katspaugh/wavesurfer.js/assets/71408982/bc99995e-3c9f-4b2c-9180-1ff075e9285e

**Comment by katspaugh** [comment]
Hmm that’s really weird. From what version did you upgrade?

**Reply by Stianlars1** [reply]
> Hmm that’s really weird. From what version did you upgrade?

I updated from 7.4.6, but it may be a problem coming from my end of it.

I tried downgrading, but it didn't work. So now I'm stuck with this problem I've never had before, and I've reset all my recent changes to a time I knew it worked, and it still doesn't. 

Kind of weird to stumble upon this issue now when I've never had it before. 

Have you ever experienced this?

**Reply by katspaugh** [reply]
Can you replicate this behavior on codepen or share a link where I can see it?
